plugins/pdr_deterministic_plugin/ufm_sim_web_service/collector.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+import urllib.error
+import logging
+from constants import PDRConstants
+class Collector:
```

Comment:
change to "TelemetryCollector"? other collected data comes from ufm_communication_mgr